libraries/AP_AccelCal/AccelCalibrator.cpp

```diff
@@ -379,9 +379,6 @@ void AccelCalibrator::run_fit(uint8_t max_iterations, float& fitness)
         }
 
         num_iterations++;
-        if (fitness - last_fitness < 1.0e-9f) {
-            break;
-        }
```

Comment:
Actually it looks like its intended for that but in reality does nothing but restricts to only one iteration since fitness is going to be < last_fitness in the first run itself and hence producing true and exiting. For Gauss Newton we anyway want to run as many iterations as we can, because there are ups and downs along the iteration so we might get say best result where before and after iterations were bad. The lines above takes care that sane and the best is selected.